### DISCUSSION TITLE
Locally hosting the interactsh-server

### DISCUSSION THREAD

**Discussion by JanHoffmannTU** [projectdiscovery/interactsh#324] [Q&A] [answered]
Hello! I am making some modifications to the interactsh-server script and would want to be able to test them locally, rather than uploading them to the server that is actually hosting it every time.
Any help on how to achieve this would be very much appreciated!

**Comment by ehsandeep** [comment, marked as answer]
You can run it locally by disabling the SSL certificate registration

```
   -sa, -skip-acme                          skip acme registration (certificate checks/handshake + TLS protocols will be disabled)
```

```
./interactsh-server -sa -d localhost
```
